readthedocs/core/views/__init__.py

```diff
@@ -84,6 +84,14 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class TeapotView(TemplateView):
+    template_name = "core/teapot.html"
+
+    def get(self, request, *args, **kwargs):
+        context = self.get_context_data(**kwargs)
+        return self.render_to_response(context, status=403)
```

Comment:
Teapot is 418: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418

Reply:
Meh, I forgot to change this. Thanks!